.golangci.yml

```diff
@@ -36,6 +36,10 @@ linters-settings:
 linters:
   enable-all: true
   disable:
+    - dupword
+    - wastedassign
```

Comment:
why ignore them?

Reply:
Wanted to keep the PR just the 1.20 bump, these are all new linters which have a bunch of failures across the code base so I think it would be good to fix them in another PR.

Reply:
sure, please open a ticket to handle them

Reply:
Created an issue for fixing this. Closing the conversation.